### ISSUE TITLE
The height of the navbar is disproportionate.

### ISSUE THREAD

**Issue by anemol** [NodeBB/nodebb-theme-persona#407]
The height of the navbar is disproportionate.

![spectacle wz7391](https://user-images.githubusercontent.com/10926763/35935692-79cdf860-0c41-11e8-8200-34cd65e3931d.png)

The logo image is a 64x64 png. Even without it, the problem is the same.

I upgraded nodebb to the last version (1.7.4) and still have this trouble.

How to fix it?

**Comment by julianlam** [comment]
:thinking: Are you using a bootswatch skin? Does the problem go away when you switch back to a skin-less setup?

**Comment by anemol** [comment]
Initially, i only applied  "Persona" theme and Cyborg (or Darkly) skin. An forum user bipped about a problem of presentation. On my side, navbar was fine but when i cleaned browser cache, i got the same  presentation as him.

For now, i have to apply "Slick theme" to prevent this ugly navbar and ask to users to apply a darktheme into their profile settings.

When i build nodebb i have this message: (maybe nothing to see with the current problem)
```
Witout `from` option PostCSS could generate wrong source map or do not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning
Witout `from` option PostCSS could generate wrong source map or do not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning


```

**Comment by barisusakli** [comment]
This is fixed on master and 1.7.x  see https://github.com/NodeBB/NodeBB/issues/6306
